### PR TITLE
Install the GPU drivers from package manager

### PR DESCRIPTION
**Pull request by arjunrn**
Install the nvidia drivers from the Ubuntu package repositories to ensure compatibility.

### CODE REVIEW

File: test.sh [hunk 1]
```diff
@@ -60,6 +60,78 @@ fi
...
+    if [ $alive -eq 0 ]; then
+        break
+    fi
+
+    sleep 2
+done
+
+terminationcounter=300
+
+while [ true ]; do
```
Comment (szuecs) [inline test.sh]: why not a for loop here?
`for i in {1..300}; do ...; done`

Comment (arjunrn) [reply test.sh]: Because I just copied a similar test and made some small changes.

Comment (arjunrn) [reply test.sh]: Also with a fixed number of attempts there will have to be error handling and in bash. This way the pipeline just times out. Also it's expected that this step won't fail.

Comment (szuecs) [reply test.sh]: true, but with a for loop you do not need to use the bash math expression and reduce the lines of code.

**Comment by aermakov-zalando** [comment]
👍

**Comment by arjunrn** [comment]
:+1:

**Pull request merged**